AndroidNativeTemplate/build.gradle

```diff
@@ -5,7 +5,8 @@ buildscript {
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:7.2.1'
+    classpath 'com.android.tools.build:gradle:7.3.1'
+    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.0"
```

Comment:
Since Kotlin isn't used in this project I figured I would add it as the min version needed, just to ensure that remains true.  Gradle in our main repo _should_ throw errors if we use new features, but I figured this couldn't hurt as an extra safety net.